### ISSUE TITLE
Permanent Virtual Machines

### ISSUE THREAD

**Issue by patrickbucher** [patrickbucher/achim#1]
It shall be possible to mark a virtual machine as permanent with a label (e.g. `type=permanent`, `permanent=true`, `lifetime=permanent` etc.). This special label shall be assigned when the virtual machine is created (using either `create-instance` or `create-group`) based on a label/tag that is set in the groups file:

```yaml
name: teachers
users:
- email: patrick.bucher@sluz.ch
  ssh-key: 'ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIF4JGW5PzpeGh1oFyultAos1RUeJSpexOaQ8odLKmVWV patrick.bucher@sluz.ch'
  name: patrick_bucher
  labels:
    - permanent: true
```

The commands `destroy-instance` and `destroy-group` must not delete virtual machines marked as permanent, unless an additional switch (e.g. `--include-permanent`) is set. The `stop-instance` and `stop-groups` commands remain the same, for permanent virtual machines must still be shutdown after usage. The `start-instance` and `start-group` commands remain the same, too.

**Comment by patrickbucher** [comment]
The concept of `labels` shall not be exposed to the configuration layer. It's an implementation detail. Therefore, the `permanent` flag shall not be a label, but an additional setting for each user:

```yaml
name: teachers
users:
  - email: patrick.bucher@sluz.ch
    name: patrick_bucher
    ssh-key: "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIJhx+5Z/Ao+9w3EujyXcKcE8Wzw0uyApawro5dlMXQwa patrick.bucher@sluz.ch"
    permanent: true
```

**Comment by patrickbucher** [comment]
Resolved in v0.0.7.
